player_rating/tests.py

```diff
@@ -66,11 +66,14 @@ def test_create_player_rating_invalid_rating2(self, player_rating):
                 rating=0)
 
     def test_delete_player_deletes_player_ratings(self, player, player_rating):
-        assert Player.objects.count() == 1
-        assert PlayerRating.objects.count() == 1
+        all_players_ratings_before = PlayerRating.objects.count()
+        player_fixture_ratings = PlayerRating.objects.filter(player=player).count()
+        assert Player.objects.filter(user=player.user).exists()
+        assert PlayerRating.objects.count() > 0
         player.delete()
-        assert Player.objects.count() == 0
-        assert PlayerRating.objects.count() == 0
+        with pytest.raises(Player.DoesNotExist):
+            Player.objects.get(pk=player.pk)
```

Comment:
There's no need to test this as it has already been verified in `test_delete_player `in the player model tests

Reply:
removed